index.html

```diff
@@ -0,0 +1,218 @@
+              <h3>Version control</h3>
+              <ul>
+                <li>Git</li>
+                <li>Github</li>
+              </ul>
+            </div>
+            <div class="data-base">
+              <h3>Database</h3>
+              <ul>
+                <li>MySQL</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- <section id="portfolio">
+        <div class="portfolio">
+          <div class="portfolio-heading">
+            <h4>Portfolio</h4>
+          </div>
+          <div class="portfolio-list">
+            <div class="portfolio_ls"></div>
+            <div class="portfolio_ls"></div>
+            <div class="portfolio_ls"></div>
+            <div class="portfolio_ls"></div>
+          </div>
+        </div>
+      </section> -->
+      <section id="contact">
+        <div class="contact">
+          <div class="contact-header">
+            <p>Let's stay connected</p>
+            <div class="socialmedia-icons">
+              <a
+                href="https://twitter.com/toyin_ayorinde"
+                target="_blank"
+                rel="noreferrer"
+              >
+                <i class="fab fa-2x fa-twitter-square"></i>
+              </a>
+              <a
+                href="https://www.linkedin.com/in/toyin-ayorinde-55a859101/"
+                target="_blank"
+                rel="noreferrer"
+              >
+                <i class="fab fa-2x fa-linkedin"></i>
+              </a>
+              <a
+                href="https://github.com/Temitoyin"
+                target="_blank"
+                rel="noreferrer"
+              >
+                <i class="fab fa-2x fa-github"></i>
+              </a>
+              <a
+                href="https://codepen.io/temitoyin"
+                target="blank"
+                rel="noreferrer"
+              >
+                <i class="fab fa-2x fa-codepen"></i>
+              </a>
+            </div>
+          </div>
+          <div class="contact-content">
+            <p>
+              I'm currently availble for web development projects. If you have a
+              wonderful idea that you are looking to implement, don't hesitate
+              to get in touch
+            </p>
+            <div class="form-container">
+              <form>
+                <label for="name">Name</label>
+                <input type="text" name="name" placeholder="Name" />
+                <label for="email">Email</label>
+                <input type="text " name="email" placeholder="Email" />
+                <label for="message">Message</label>
+                <textarea id="message" rows="4" cols="50" placeholder="Message">
+                </textarea>
+                <button>Contact</button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-copyright">Temitoyin © 2020 Copyright</div>
```

Comment:
You can try making this date dynamic using a little bit of Javascript. That way you don't have to update this every new year. Do you want to give this a shot?

Reply:
Done